tests/ui/test_ui_tools.py

```diff
@@ -1896,20 +1896,56 @@ def test_keypress_STREAM_MESSAGE(self, mocker, msg_box, widget_size,
                               'realm_editing_allowed',
                               'msg_body_edit_enabled',
                               'expect_editing_to_succeed'], [
-        ({'sender_id': 2, 'timestamp': 45}, True, True, False),
-        ({'sender_id': 1, 'timestamp': 1}, True, False, True),
-        ({'sender_id': 1, 'timestamp': 45}, False, True, False),
-        ({'sender_id': 1, 'timestamp': 45}, True, True, True),
+        ({
+            'sender_id': 2,
+            'timestamp': 45,
+            'subject': 'abc'
+         }, True, True, False),
+        ({
+            'sender_id': 1,
+            'timestamp': 1,
+            'subject': 'hello'
+         }, True, False, True),
+        ({
+            'sender_id': 1,
+            'timestamp': 45,
+            'subject': 'zt'
+         }, False, True, False),
+        ({
+            'sender_id': 1,
+            'timestamp': 1,
+            'subject': '(no topic)'
+         }, True, False, True),
+        ({
+            'sender_id': 2,
+            'timestamp': 1,
+            'subject': '(no topic)'
+         }, True, False, True),
+        ({
+            'sender_id': 1,
+            'timestamp': 1,
+            'subject': '(no topic)'
+         }, False, True, False),
+        ({
+            'sender_id': 1,
+            'timestamp': 45,
+            'subject': 'intro'
+         }, True, True, True),
     ], ids=['msg_sent_by_other_user',
             'topic_edit_only_after_time_limit',
             'editing_not_allowed',
-            'all_conditions_met'])
+            'msg_sent_by_me_with_no_topic',
+            'msg_sent_by_other_with_no_topic',
+            'editing_not_allowed_for_no_topic',
+            'all_conditions_met', ])
     def test_keypress_EDIT_MESSAGE(self, mocker, message_fixture, widget_size,
                                    expect_editing_to_succeed,
                                    to_vary_in_each_message,
                                    realm_editing_allowed,
                                    msg_body_edit_enabled,
                                    key):
+        if message_fixture['type'] == 'private':
+            to_vary_in_each_message['subject'] = ''
```

Comment:
Is this what we expect in PMs, or is the key absent?

Reply:
The API docs do say that we should expect an empty string here. Quote:
> subject: string The topic of the message. Currently always "" for private messages, though this could change if Zulip adds support for topics in private message conversations.

I also did a manual test just to be extra sure. :)